.github/workflows/ruby_test.yml

```diff
@@ -201,6 +201,7 @@ jobs:
     runs-on: ubuntu-latest
     needs: [rake_tests, rspec_tests, cucumber_tests]
     steps:
+    - uses: actions/checkout@v2
     - name: Fetch coverage results
```

Comment:
Need the code checked out as the report uploaded needs access to the git repo for commits/branches etc.